department-of-veterans-affairs/caseflow
# Resolved bugs on hearing details page when changing timezone and rece…

Resolves 1. and 2. of #14840

### Description
Fixes issues found on the hearing details page discovered during the dogfooding session

### Acceptance Criteria
- [ ] All reported bugs are resolved

### Testing Plan
1. First you will need to comment line 83 of the `app/serializers/hearings/hearing_serializer.rb` file
1. Next shadow user `BVASYELLOW`
1. Navigate to the hearings schedule and choose a Central hearing: http://localhost:3000/hearings/schedule
1. Click the 'Edit Hearing Details' link
1. Change the type to Virtual
1. Ensure that you receive an error prompting you to input a timezone
1. Add the timezone and ensure that you can save
1. Without refreshing the page, scroll down and ensure that you see the email events
